yb-voyager/cmd/importData.go

```diff
@@ -500,19 +499,21 @@ func splitDataFiles(importTables []string, taskQueue chan *SplitFileImportTask)
 		extractCopyStmtForTable(t, origDataFile)
 		log.Infof("Start splitting table %q: data-file: %q", t, origDataFile)
 
-		log.Infof("Collect interrupted splits.")
+		log.Infof("Collect all interrupted/remaining splits.")
 		largestCreatedSplitSoFar := int64(0)
 		largestOffsetSoFar := int64(0)
 		fileFullySplit := false
 		pattern := fmt.Sprintf("%s/%s/data/%s.[0-9]*.[0-9]*.[0-9]*.[CPD]", exportDir, metaInfoDir, t)
 		matches, _ := filepath.Glob(pattern)
-		// in progress are interrupted ones
-		interruptedRegexStr := fmt.Sprintf(".+/%s\\.(\\d+)\\.(\\d+)\\.(\\d+)\\.(\\d+)\\.[P]$", t)
-		interruptedRegexp := regexp.MustCompile(interruptedRegexStr)
+
+		doneSplitRegexStr := fmt.Sprintf(".+/%s\\.(\\d+)\\.(\\d+)\\.(\\d+)\\.(\\d+)\\.[D]$", t)
+		doneSplitRegexp := regexp.MustCompile(doneSplitRegexStr)
+		validTableSplitRegexStr := fmt.Sprintf(".+/%s\\.(\\d+)\\.(\\d+)\\.(\\d+)\\.(\\d+)\\.[CPD]$", t)
```

Comment:
How about naming it as `regexSplitFileName`?

Reply:
Can name it as `splitFileNameRegexStr`
currently, all other similar variables names ends with `Regex`, Just following the same pattern here. We can consider renaming to this pattern all variables later.

Reply:
Is that fine if i name `splitFileNameRegexStr` for now?

Reply:
Yes. You can use the name `splitFileNameRegexStr`.